CHANGELOG.md

```diff
@@ -29,6 +29,16 @@ instructions](https://nasa.github.io/cumulus/docs/upgrade-notes/upgrade-rds-clus
   - Update `cumulus-rds-tf` module to take additional parameters in support of
     migration from Aurora PostgreSQl v11 to v13.   See Migration Notes for more details.
 
+## Changed
```

Comment:
This isn't right. There's a duplicate 'Changed' heading under 'unreleased' and we're missing the correct entry under 16.1.4.

Reply:
The screenshot looks correct but make sure the formatting is correct. Preview the markdown in vscode and make sure everything looks accurate. Also double-check for duplicate headings etc.

Reply:
```suggestion

```